### DISCUSSION TITLE
Data Egress via supavisor double in a Saturday

### DISCUSSION THREAD

**Discussion by Nicolasvegam** [supabase/supabase#28605] [Questions]
Hi guys!

We running a small startup, and our project is not that big.

Last 31th of July our data egress peaked! And starting from that day, the consume is twice as big!

We have no more info or logs in order to see what happened, also in the supabase billing we couldn't see more details

I already sent a ticket in order to get help

**Current Billing**

<img width="1074" alt="image" src="https://github.com/user-attachments/assets/3f03942d-c3e9-40f7-bce0-7f166bf0bf90">

<img width="1098" alt="image" src="https://github.com/user-attachments/assets/f91d2138-1bd2-40f3-9bee-c8947a4053b7">

<img width="639" alt="image" src="https://github.com/user-attachments/assets/49b7e44d-9597-484f-931c-add569f4501c">

**Last Billing**

<img width="1081" alt="image" src="https://github.com/user-attachments/assets/154d70f5-44e4-4094-ae16-a1127374d661">

**Server logs**

<img width="1171" alt="image" src="https://github.com/user-attachments/assets/682f8291-93ff-4681-8cb7-50794f9e3711">

Any ideas in order to debug this problem?

**Comment by GaryAustin1** [comment]
Is red realtime?

**Reply by Nicolasvegam** [reply]
Not, is supavisor egress

**Reply by Nicolasvegam** [reply]
updated*

**Reply by GaryAustin1** [reply]
(Not sure how the colors are picked... on mine realtime is red is why I asked).

What do you have using the direct/pooled database ports?

**Reply by Nicolasvegam** [reply]
actually, just my server (nest js backend), which doesn't have peaks (one but on October 8th)

**Comment by flazouh** [comment]
I have the same problem, did you solve it eventually ? I'm following this guide ... https://github.com/orgs/supabase/discussions/26605 @Nicolasvegam
